spec/middleman-prismic/commands/prismic_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "spec_helper"
+
+describe Middleman::Cli::Prismic do
+  describe "#prismic" do
+    it "creates the directory that prismic data should live in" do
+      stub_options
+      stub_prismic_api
+
+      within_temp_directory do
+        expect(File).to_not be_exists(described_class::DATA_DIR)
+
+        described_class.new.prismic
+
+        expect(File).to be_exists(described_class::DATA_DIR)
+      end
+    end
+
+    it "collects all available documents despite pagination and writes them to file" do
+      stub_options
+      document = double("document", id: "thing", to_yaml: "yaml")
+      response = double(
+        "response",
+        group_by: { "foo" => [document] },
+        total_pages: 1,
+      )
+      stub_prismic_api(response)
+
+      within_temp_directory do
+        described_class.new.prismic
+
+        file_path = File.join(described_class::DATA_DIR, "foos")
+
+        expect(File).to be_exists(file_path)
+        expect(Dir.children(file_path)).
+          to eq ["#{Digest::MD5.hexdigest(document.id)}.yml"]
```

Comment:
why `hexdigest`? i don't see this referenced anywhere else, including in the rest of this codebase. maybe a clarifying comment or, better yet, method or class could be helpful to provide some clarity